_includes/partners/contact.html

```diff
@@ -0,0 +1,35 @@
+<main id="main-content">
+  <section>
+    <div class="container partners-container">
+      <div class="grid-row">
+        <div class="tablet:grid-col-9 accordion-container">
+          <script charset="utf-8" type="text/javascript" src="//js.hsforms.net/forms/shell.js"></script>
+          <script>
```

Comment:
I guess the only scripting language these days is JS...but I think it's good to be explicit anyways
```suggestion
          <script type="text/javascript" >
```

Reply:
Hubspot gives the snippet, which I do not normally tweak much myself.